libs/components/src/lib/side-drawer/side-drawer.ts

```diff
@@ -42,4 +42,21 @@ export class SideDrawer extends FoundationElement {
 	@attr({
 		mode: 'boolean',
 	}) trailing = false;
+
+	override attributeChangedCallback(name: string, oldValue: string, newValue: string): void {
+		super.attributeChangedCallback(name, oldValue, newValue);
+		switch (name) {
+			case 'open': {
+				this.open ? this.#opened() : this.#closed();
+			}
+		}
+	}
+
+	#closed(): void {
+		this.$emit('closed');
```

Comment:
if we trigger events on start, we should name them as simple present tense, right?
```suggestion
		this.$emit('close');
```